setup.py

```diff
@@ -24,7 +24,7 @@ def package_files(directory, strip_leading):
     long_description = fh.read()
 
 setup(name='donkeycar',
-      version="4.3.10",
+      version="4.3.11",
```

Comment:
I think this needs to be 4.3.14 now - don't know why GitHub doesn't show it as conflict.